### PR TITLE
Add form + table view, add prettier, style with Pico CSS

### PR DESCRIPTION
**Pull request by AndreasPB**

### CODE REVIEW

File: api/app/main.py [hunk 1]
```diff
@@ -1,9 +1,18 @@
 from fastapi import FastAPI
```
Comment (AndreasPB) [inline api/app/main.py:1]: Remove backend stuff from here - It was just to test

**Comment by AndreasPB** [comment]
Just a heads-up, I ran `pre-commit --all-files` with the new pre-commit settings, so that's why a lot of the files are getting indented/whatever

File: api/app/main.py [hunk 2]
```diff
@@ -1,22 +1,40 @@
...
+async def post_user(user: UserCreate):
+    """Posts a user"""
     await User(**user.dict()).save()
     return user
 
 
-@app.get("/find-user/{name}")
-async def _(name: str):
+@app.get("/user")
+async def find_users():
```
Comment (nullxDEADBEEF) [inline api/app/main.py]: Ligesom med post_user, lad os holde os til REST navnene(get_user, post_user, delete_user, etc.)

File: api/app/main.py [hunk 3]
```diff
@@ -1,22 +1,40 @@
...
+    """Finds all users"""
+    users = await User.find().to_list()
+    return list(map(dict, users))
+
+
+@app.get("/user/{name}")
+async def find_user_by_name(name: str):
```
Comment (nullxDEADBEEF) [inline api/app/main.py]: Samme som over

**Review by nullxDEADBEEF — CHANGES_REQUESTED** [review]

File: api/app/main.py (hunk 2 above)
Comment (AndreasPB) [reply api/app/main.py]: Done

File: api/app/main.py (hunk 3 above)
Comment (AndreasPB) [reply api/app/main.py]: Done

File: api/app/main.py (hunk 1 above)
Comment (AndreasPB) [reply api/app/main.py:1]: I/we decided tp just leave it... IDK

**Review by nullxDEADBEEF — APPROVED** [review]
:rocket: :guitar:

**Pull request merged**